doc/configuring.rst

```diff
@@ -436,3 +436,68 @@ If executing the build step depends on more than one environment variable, use `
 For example, ``if_env_set="SPECIAL_TOOL_PATH && ADDITIONAL_SOURCES_ROOT"`` step will be executed only
 in case of both `$SPECIAL_TOOL_PATH` and `$ADDITIONAL_SOURCES_ROOT` environment variables set to some values.
 If any of them is missing or not set in current environment, the step will be excluded from current run.
+
+
+Conditional steps
+---------------------
+
+Conditional step - :class:`Step`, that has ``if_succeeded`` and/or ``if_failed`` parameters with other steps assigned.
+If the conditional step will succeed, then the step from the ``if_succeeded`` parameter will be executed.
+If the conditional step will fail, the step from the ``if_failed`` parameter will be executed instead.
+
+Configuration example:
+
+.. testcode::
+
+    from universum.configuration_support import Configuration, Step
+
+    true_branch_step = Step(name="Positive branch step", command=["ls"])
+    false_branch_step = Step(name="Negative branch step", command=["pwd"])
+    conditional_step = Step(name="Conditional step", command=["./script.sh"],
+                            if_succeeded=true_branch_step, if_failed=false_branch_step)
+
+    configs = Configuration([conditional_step])
+
+If ``script.sh`` will return zero exit code:
```

Comment:
"Here's the example output for 'script.sh' returning zero exit code:"
Otherwise it's not quite obvious that quote beneath is what happens, _if._